docs/data-masking-function-list.md

```diff
@@ -66,6 +109,10 @@ mysql> SELECT gen_blocklist('apple', 'fruit', 'nut');
 
 Returns a term from a dictionary selected at random.
 
+### Version update
+  
+Percona Server for MySQL 8.0.41 introduces an internal term cache. The server now uses in-memory data structures for lookups instead of querying the `mysql.masking_dictionaries` table every time. This improvement boosts performance, especially when handling multiple rows.
```

Comment:
```suggestion
Percona Server for MySQL 8.0.41 introduces an internal term cache. The server now uses in-memory data structures for lookups instead of querying the `<masking_functions.masking_database>.masking_dictionaries` table every time. This improvement boosts performance, especially when handling multiple rows.
```